src/server.js

```diff
@@ -17,10 +18,21 @@ try {
     access_token: process.env.ACCESS_TOKEN,
     access_token_secret: process.env.ACCESS_TOKEN_SECRET,
   });
+  stream = twitBot.stream('statuses/filter', { track: 'Want to read more or fix an error? Visit Nkọwa okwu:' });
 } catch (err) {
   twitBot = defaultTwitBot;
 }
 
+stream.on('tweet', (tweet) => {
+  const id = tweet.id_str;
+  const replyCount = tweet.reply_count;
+  const word = tweet.text.split('\n')[1];
+
+  if (tweet.user.id_str === process.env.BOT_ACCOUNT_ID && replyCount === 0) {
```

Comment:
You'll have to add the `id_str` of @nkowaokwu account as an environment variable if this gets deployed.

Reply:
thanks for drawing my attention 👍🏾